src/monitoring_service/cli.py

```diff
@@ -4,20 +4,55 @@
 from typing import Any, Optional
 
 import click
+import gevent
 import structlog
 from eth_account import Account
 from eth_utils import is_checksum_address
 from web3 import HTTPProvider, Web3
 
-from monitoring_service.constants import DEFAULT_REQUIRED_CONFIRMATIONS
+from monitoring_service.constants import (
+    DEFAULT_GAS_BUFFER_FACTOR,
+    DEFAULT_GAS_CHECK_WAIT,
+    DEFAULT_REQUIRED_CONFIRMATIONS,
+)
 from monitoring_service.service import MonitoringService
+from raiden_contracts.constants import GAS_REQUIRED_FOR_MS_MONITOR
 from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
 from raiden_libs.logging import setup_logging
 from raiden_libs.types import Address
+from raiden_libs.utils import private_key_to_address
+
 
 log = structlog.get_logger(__name__)
 
 
+def check_gas_reserve(web3: Web3, private_key: str) -> None:
+    """ Check periodically for gas reserve in the account """
+    gas_price = web3.eth.gasPrice
+    gas_limit = GAS_REQUIRED_FOR_MS_MONITOR
+    estimated_required_balance = gas_limit * gas_price * DEFAULT_GAS_BUFFER_FACTOR
+    estimated_required_balance_eth = Web3.fromWei(estimated_required_balance, 'ether')
+    current_balance = web3.eth.getBalance(private_key_to_address(private_key))
+    if current_balance < estimated_required_balance:
+        log.info('Missing gas reserve', required_wei=estimated_required_balance)
+        log.error(
+            (
```

Comment:
you can put this in the last line

Reply:
I am not sure if I understand. The `log.error` is already in the last line of that function, right?

Reply:
you can just move up the `(` in the last liune

Reply:
```
        log.error(
            (
                "Your account's balance is below the estimated gas reserve of "
                f"{estimated_required_balance_eth} Eth. You will be be unable "
                "to perform on-chain transactions and cannot monitor any channels. "
                "Please add funds to your account as soon as possible."
            ),
            fg='red',)
```

Reply:
sorry ich steh grad auf dem Schlauch tbh ...

Reply:
exactly what I meant, even though i wonder if it's actually better now that I see it.

Reply:
done